modal/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "9567cd54-7fac-46d8-84ce-0b00cef6937e",
+  "app_id": "modal",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Troubleshooting",
+    "changelog": "CHANGELOG.md",
+    "description": "Collect logs and metrics for your Modal applications",
+    "title": "Modal",
+    "media": [
+      {
+        "media_type": "image",
```

Comment:
We recommend also adding an image of your Datadog dashboard

Reply:
For this image, we ask as well that you display dummy data in the dashboard to avoid the No Data state

Reply:
Done.